src/components/CollaborationBar/CollaborationBar.test.tsx

```diff
@@ -103,4 +104,39 @@ describe('<CollaborationBar>', () => {
       }),
     ).toBeInTheDocument();
   });
+
+  it('should show cursors in presentation mode if edit mode is enabled', async () => {
+    setPresentationMode(true, true);
+
+    render(<CollaborationBar />, { wrapper: Wrapper });
+
+    await userEvent.click(
+      screen.getByRole('checkbox', {
+        name: "Show collaborators' cursors",
+        checked: false,
+      }),
+    );
+
+    expect(
+      screen.getByRole('checkbox', {
+        name: "Hide collaborators' cursors",
+        checked: true,
+      }),
+    ).toBeInTheDocument();
+  });
+
+  it('should hide cursors in presentation mode if edit mode is not enabled', async () => {
+    render(<CollaborationBar />, { wrapper: Wrapper });
+
+    const toggleCollaboratorsCursors = screen.getByRole('checkbox', {
+      name: "Show collaborators' cursors",
+      checked: false,
+    });
+
+    expect(toggleCollaboratorsCursors).toBeInTheDocument();
+
+    setPresentationMode(true, false);
+
+    expect(toggleCollaboratorsCursors).not.toBeInTheDocument();
+  });
```

Comment:
Currently this test should be improved. In presentation mode if edit is not enabled we should have the following true:  there should be no "Show collaborators' cursors" checkbox and no "Hide collaborators' cursors" checkbox as well. Test checks "Show collaborators' cursors" but not checks that "Hide collaborators' cursors" is not here, please change that. In addition I don't think we need rechecking that element is not in the document, we could just set `setPresentationMode(true, false);` in the start of the test and check that both checkboxes are not rendered.